docs/api.md

```diff
@@ -544,15 +544,20 @@ const pinoOptions = {
 }
 
 function logMethod (args, method) {
-  if (args.length === 2) {
-    args[0] = `${args[0]} %j`
+  if (typeof args[0] === "string" && args.length >= 2) {
+    args[0] += ", %j".repeat(args.length - 1);
```

Comment:
This does not match the lead in text. The lead in very specifically shows two arguments.

Reply:
I don't understand your response. What is lead?

Also, I'm not aware of previous pino versions of what the original snippet tried to solve.

Reply:
https://dictionary.cambridge.org/us/dictionary/english/lead-in

"something that introduces something else, such as the words and music that are used to introduce a television program"

The example you are modifying in this PR is not in isolation. The text before it describes what is being shown in the example. The change in this PR makes this example not match the lead-in text.

Reply:
The lead in text you indicate is following:
> Since pino v6, we do not automatically concatenate and cast to string consecutive  parameters:
>
> logger.info('hello', 'world')
>

there's nowhere written in there that only two parameters must exist, consecutive means unspecified amount. so the code **does** match!

@jsumners what do you want from me I don't understand?

Reply:
The example is showing how one would replicate a very specific case that worked prior to v6. It is not a general example for a variadic implementation.

I see two possibilities:

1. Drop the change
2. Finish the PR by making the lead-in text match the example